ci/matrix.xml

```diff
@@ -132,7 +151,11 @@
         'set scalaJSOptimizerOptions in testSuite ~= (_.withDisableOptimizer(true))' \
         'set scalaJSStage in Global := FastOptStage' \
         ++$scala testSuite/test \
-        testSuite/clean &&
+        testSuite/clean
```

Comment:
Why should we do it at all? IIUC, the cleans where there, when options changed. So at the end of the task, they don't seem to make sense at all.